starport/cmd/network_request_verify.go

```diff
@@ -0,0 +1,114 @@
+package starportcmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/pkg/chaincmd"
+	"github.com/tendermint/starport/starport/pkg/clispinner"
+	"github.com/tendermint/starport/starport/pkg/numbers"
+	"github.com/tendermint/starport/starport/services/network"
+	"github.com/tendermint/starport/starport/services/network/networkchain"
+)
+
+// NewNetworkRequestVerify creates a new request approve
+// command to approve requests for a chain.
+func NewNetworkRequestVerify() *cobra.Command {
+	c := &cobra.Command{
+		Use:     "verify [launch-id] [number<,...>]",
+		Aliases: []string{"accept"},
```

Comment:
maybe the `accept` alias can be confused with the `approve` command

Reply:
This is a copy paste mistake